staging/src/k8s.io/apiserver/pkg/authorization/authorizerfactory/delegating.go

```diff
@@ -36,12 +37,28 @@ type DelegatingAuthorizerConfig struct {
 	// DenyCacheTTL is the length of time that an unsuccessful authorization response will be cached.
 	// You generally want more responsive, "deny, try again" flows.
 	DenyCacheTTL time.Duration
+
+	// SkippedPaths are HTTP paths which are excluded from authentication. They can be plain
+	// paths or end in * in which case prefix-match is applied. A leading / is optional.
```

Comment:
not a fan of making the input here flexible... what was the reason around that?

Reply:
`/healthz/*`, what would you expect? Just fixed paths? I.e. without the star option?